astropy/io/ascii/latex.py

```diff
@@ -310,16 +276,14 @@ def __init__(self, ignore_latex_commands=['hline', 'vspace', 'tableline'],
         self.data.comment = self.header.comment
 
     def write(self, table=None):
-        self.header.start_line = None
-        self.data.start_line = None
+        self.header.start_line = 0
+        self.data.start_line = 0
```

Comment:
Could you provide some information why you changed these lines? I haven't read all of the related discussion. But you don't use these variables later on.